instagram.user.styl

```
@@ -105,13 +110,26 @@ rad() { border-radius: arguments }
         --cyan      : _cyan;
         --green     : _green;
 
+        // Enable theme scrollbar width
+        if ( custom_width == 1) {
+            /::-webkit-scrollbar {
+                width: .5em
+            }
+        }
+
         /// Enable theme scrollbars.
-        if ( enable_sb == 1 ) {
-            scrollbar-color: #88888840 #88888815;
-            ::-webkit-scrollbar {
+        if ( sb_color == "dark" ) {
+            /html { scrollbar-color: #88888840 #88888815; }
+            /::-webkit-scrollbar {
                 c: 0 0 #88888815;
                 &-thumb { c: 0 0 #88888840 }
             }
+        } else if ( sb_color == "accent" ) {
+            /html { scrollbar-color: #7289da transparent; }
```

It probably needs to be added to line 132 as well.

```suggestion
            /html { scrollbar-color: #7289da #88888815; }
```